### ISSUE TITLE
​PDF not properly got uploaded, it's not opening properly

### ISSUE THREAD

**Issue by bigint** [toeverything/AFFiNE#6206]
### What happened?

​PDF not properly got uploaded, it's not opening properly

https://github.com/toeverything/AFFiNE/assets/69431456/b8ce74a6-8331-4750-8e6f-8982faac415f

### Distribution version

Web (insider.affine.pro)

### What browsers are you seeing the problem on if you're using web version?

Chrome

### Are you self-hosting?

- [ ] Yes

**Comment by CatsJuice** [comment]
Yes, PDF previewer is not implemented yet, and we have already planned this, it won't take too much time.
